pages/products/index.tsx

```diff
@@ -0,0 +1,187 @@
+import { ArrowRightIcon } from "@radix-ui/react-icons";
+import Image from "next/image";
+import ButtonLink from "../../components/ButtonLink";
+
+const Container = styled("section", {
+  pt: "$12",
+});
+
+const HeroContainer = styled("div", {
+  display: "flex",
+  flexDirection: "column",
+  justifyContent: "space-between",
+  height: 200,
+  position: "relative",
+  h1: {
+    position: "absolute",
+    left: 4,
+    top: "calc(50% - 22px)",
+    fontSize: 32,
+    lineHeight: "44px",
+    color: "$forest",
+    mt: 0,
+  },
+});
+
+const Row = styled("div", {
+  display: "flex",
+  justifyContent: "space-between",
+});
+
+const HeroElement = styled("div", {
+  marginTop: 2.31,
+  width: 8,
+  height: 4.62,
+  backgroundColor: "$sprout",
+  position: "relative",
+  "&:before": {
+    content: "",
+    borderLeft: "4px solid transparent",
+    borderRight: "4px solid transparent",
+    position: "absolute",
+    top: -2.31,
+    borderBottom: "2.31px solid $sprout",
+  },
+  "&:after": {
+    content: "",
+    borderLeft: "4px solid transparent",
+    borderRight: "4px solid transparent",
+    position: "absolute",
+    bottom: -2.31,
+    borderTop: "2.31px solid $sprout",
+  },
+});
+
+const InnerHeroElement = styled(HeroElement, {
+  transform: "scale(0.7,0.7)",
+  backgroundColor: "$sand",
+  zIndex: 1,
+  marginTop: 0,
+  "&:before": {
+    borderBottomColor: "$sand",
+  },
+  "&:after": {
+    borderTopColor: "$sand",
+  },
+});
+
+const ProductsHero = () => (
+  <Container>
+    <Wrapper>
+      <HeroContainer>
+        {Array(4).fill(
+          <Row>
+            {Array(17).fill(
+              <HeroElement>
+                <InnerHeroElement />
+              </HeroElement>
+            )}
+          </Row>
+        )}
+        <h1>Products built by Cabin</h1>
+      </HeroContainer>
+    </Wrapper>
+  </Container>
+);
+
+const PRODUCTS = [
+  {
+    name: "NFT Passports",
+    description:
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud.",
+    path: "passports",
+  },
+];
+
+const ProductInfoContainer = styled("section", {
+  pt: "$20",
+});
+
+const ProductInfoInnerContainer = styled("section", {
+  display: "flex",
+  justifyContent: "space-between",
+  alignItems: "center",
+});
+
+const ProductImageContainer = styled("div", {
+  textAlign: "end",
+  minWidth: "50%",
+});
+
+const ProductInfo = (props: typeof PRODUCTS[number]) => {
+  return (
+    <ProductInfoContainer>
+      <Wrapper>
+        <ProductInfoInnerContainer>
+          <div>
+            <h1>{props.name}</h1>
+            <p>{props.description}</p>
+            <Link href={`/products/${props.path}`} passHref>
+              <a>
+                <ArrowRightIcon /> Read More
+              </a>
+            </Link>
+          </div>
+          <ProductImageContainer>
+            <Image
+              src={`/products/${props.path}.png`}
```

Comment:
could make this a pattern of storing images so that it's easily discoverable here

Reply:
Yeah, also open to ideas on how we can better manage and maintain content/images in the future. At some point I assume it will make sense to have a light CMS or something.

Reply:
yea in general it would be better to dynamically fetch this data as part of `getStaticProps` instead of hardcoding here. Haven't used any solutions myself but I'm sure one with Next/Vercel integration exists